.pylintrc

```diff
@@ -195,7 +195,7 @@ indent-string='    '
 [SIMILARITIES]
 
 # Minimum lines number of a similarity.
-min-similarity-lines=4
```

Comment:
Don't do this. Simply add `#pylint: disable=duplicate-code` to the duplicated block in your test.